src/main/java/com/sap/cloudfoundry/client/facade/domain/CloudRoute.java

```diff
@@ -35,10 +30,6 @@ public String getName() {
         return getHost() + "." + getDomain();
     }
 
-    public boolean isUsed() {
-        return getAppsUsingRoute() > 0 || hasServiceUsingRoute();
-    }
-
```

Comment:
This is used in the multiapps-controller, isn`t it?

Reply:
It's not